### ISSUE TITLE
Warning from jointRRR

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1116]
**Reported by edo&#46;drenth on 21 May 2013 11:58 UTC**
Checking the PlanarLoops_analytic model in Dymola 2014 creates warnings of the following format:

The following parameters with fixed = false also have a binding:
jointRRR1.n_b = {0, 0, 1}
jointRRR2.n_b = {0, 0, 1}
jointRRR3.n_b = {0, 0, 1}

Is this correct?

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1116

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 22 May 2013 07:00 UTC**
This is strange. I do not get this warning message with Dymola 2014 (tried with pedantic check on/off, Evaluate on/off, on Modelica.Mechanics.MultiBody.Joints.Assemblies.JointRRR, and on Modelica.Mechanics.MultiBody.Examples.Loops.PlanarLoops_analytic). Please, provide the model where this message occurs.

I checked the code:

```
  final parameter Real n_b[3](
    each final unit="1",
    each fixed=false,
    start={0,0,1})
initial equation    
     n_b = Frames.resolve2(frame_b.R, Frames.resolve1(frame_a.R, n_a));

```
and all this is correct Modelica ("start" is a guess value since fixed = false, in case nonlinear loops n_b occur).

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 22 May 2013 07:23 UTC**
The error was reported for MSL 3.2 not the development version. There the error is still present but has been fixed for MSL 3.2.1 by adding a missing `each`. Unfortunately the fix was not documented so it's hard to determine the exact revision when it was fixed.

**Comment by modelica-trac-importer** [comment]
**Changelog modified by dietmarw on 22 May 2013 07:23 UTC**
Initialisation of `jointRRR` corrected by adding a missing `each`.
